### ISSUE TITLE
Not reading filename reference links in chat conversation by screen reader

### ISSUE THREAD

**Issue by adamsamec** [microsoft/vscode#312769]
### Description

Whenever a GitHub copilot response contains a filename in the form of file reference link, screen reader completely ignores such links and does not narrate anything in place of the link. Screen reader user thus has no way to find out the filenames contained in the response let alone navigate to the file reference.

### Repro steps

1. Open some folder within a git repository in Visual Studio code. Choose a git repository which has some commits.
2. Turn on screen reader, such as Narrator, NVDA or JAWS.
3. Send the following prompt to GitHub Copilot: "Please give me the changes in the last commit including changed files references"
4. Navigate to the GitHub Copilot chat conversation listbox (Control + Up arrow from the chat input field).
5. Navigate to the latest response from GitHub Copilot using the Down arrow key.
6. Review the response in the Accessible viewer using the Alt + F2 keyboard shortcut.
7. Navigate the response line by line using the Down arrow key and listen how screen reader narrates filename references in the response.

### Actual result

The screen reader does not read the filename references in the response at all. The bulletted list of filenames is empty.

### Expected results

The filename references in the response should be read by the screen reader as links pointing to the respective file and possibly to a specific line in the file.
